Test failure looks unrelated. Kicking build again
```python
ERROR: testArchive (couchforms.tests.test_archive.TestFormArchiving)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "corehq/ex-submodules/couchforms/tests/test_archive.py", line 26, in testArchive
    xml_data = self.get_xml('basic')
  File "/home/travis/build/dimagi/commcare-hq/corehq/util/test_utils.py", line 60, in get_xml
    return cls.get_file(name, 'xml', override_path)
  File "/home/travis/build/dimagi/commcare-hq/corehq/util/test_utils.py", line 46, in get_file
    with open(cls.get_path(name, ext, override_path)) as f:
IOError: [Errno 2] No such file or directory: 'corehq/ex-submodules/couchforms/tests/data/xforms/basic.xml'

--- a/corehq/ex-submodules/couchforms/tests/data/sample_xforms/basic.xml
+++ b/corehq/ex-submodules/couchforms/tests/data/sample_xforms/basic.xml
@@ -1,0 +1,18 @@
+<?xml version='1.0' ?>
+<data uiVersion="1" version="17" name="New Form" xmlns:jrm="http://dev.commcarehq.org/jr/xforms" xmlns="http://openrosa.org/formdesigner/1DFD8610-91E3-4409-BF8B-02D3B4FF3530">
+    <foo>bar</foo>
+    <n0:case case_id="ddb8e2b3-7ce0-43e4-ad45-d7a2eebe9169" user_id="user-xxx-eve" date_modified="2013-04-19T16:53:02.799-04" xmlns:n0="http://commcarehq.org/case/transaction/v2">
+        <n0:update>
+            <n0:foo> bar</n0:foo>
+        </n0:update>
+    </n0:case>
+    <n1:meta xmlns:n1="http://openrosa.org/jr/xforms">
+        <n1:deviceID>A00000245706EE</n1:deviceID>
+        <n1:timeStart>2013-04-19T16:52:41.000-04</n1:timeStart>
+        <n1:timeEnd>2013-04-19T16:53:02.799-04</n1:timeEnd>
+        <n1:username>eve</n1:username>
+        <n1:userID>user-xxx-eve</n1:userID>
+        <n1:instanceID>b58df19c-efd5-4ecf-9581-65dda8b8787c</n1:instanceID>
+        <n2:appVersion xmlns:n2="http://commcarehq.org/xforms">CommCare ODK, version "2.4.1"(10083). App v19. CommCare Version 2.4. Build 10083, built on: March-12-2013</n2:appVersion>
+    </n1:meta>
+</data>

--- a/corehq/ex-submodules/couchforms/tests/test_archive.py
+++ b/corehq/ex-submodules/couchforms/tests/test_archive.py
@@ -11,7 +11,7 @@ from corehq.util.test_utils import TestFileMixin
 
 
 class TestFormArchiving(TestCase, TestFileMixin):
-    file_path = ('data', 'xforms')
+    file_path = ('data', 'sample_xforms')
     root = os.path.dirname(__file__)
 
     @classmethod

